<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">font-size: 13pt; background-image: url(:/wp/wp.png) </string>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>361</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #111111; color: #869BC0; border-radius: 20px;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="saveKeys">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>321</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #202020; border: none; </string>
     </property>
     <property name="text">
      <string>Uložit klíče</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">;border: 0</string>
     </property>
     <property name="text">
      <string>Generování klíčů</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>361</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #111111; color: #99C27D; border-radius: 20px;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="podpisOpenPrivateKey">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>155</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #202020; border: none</string>
     </property>
     <property name="text">
      <string>Otevřít privátní klíč</string>
     </property>
    </widget>
    <widget class="QPushButton" name="podpisOpenFile">
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>230</y>
       <width>155</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #202020; border: none</string>
     </property>
     <property name="text">
      <string>Otevřít soubor</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">;border: 0</string>
     </property>
     <property name="text">
      <string>Podpis</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="podpisSaveFile">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>321</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #202020; border: none</string>
     </property>
     <property name="text">
      <string>Uložit soubor</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="podpisFileInfoD">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>221</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ECBFBD;border: 0</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="podpisFileInfoP">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>81</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #C6AAE8;border: 0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>190</y>
      <width>361</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #111111; color: #B8727B; border-radius: 20px; </string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="overeniOpenPublicKey">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>155</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #202020; border: none</string>
     </property>
     <property name="text">
      <string>Otevřít veřejný klíč</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="overeniFileInfoP">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>81</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #C6AAE8;border: 0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="overeniOpenFile">
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>230</y>
       <width>155</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #202020; border: none</string>
     </property>
     <property name="text">
      <string>Otevřít soubor</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">;border: 0</string>
     </property>
     <property name="text">
      <string>Ověření</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="overeniOpenSignature">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>321</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: #202020; border: none</string>
     </property>
     <property name="text">
      <string>Otevřít podpis</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="overeniFileInfoD">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>221</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ECBFBD;border: 0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>361</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #111111; color: #869BC0; border-radius: 20px;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTextBrowser" name="message">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>10</y>
       <width>331</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="assets/wp.qrc"/>
 </resources>
 <connections/>
</ui>
